aspnetcore/log-mon/metrics/metrics.md

```diff
@@ -0,0 +1,174 @@
+---
+title: ASP.NET Core metrics
+description: Metrics for ASP.NET Core apps
+author: rick-anderson
+ms.author: riande
+monikerRange: '>= aspnetcore-8.0'
+ms.date: 6/30/2023
+ms.topic: article
+ms.prod: aspnet-core
+uid: log-mon/metrics/metrics
+---
+
+# ASP.NET Core metrics
+
+Metrics are numerical measurements reported over time. They're typically used to monitor the health of an app and generate alerts. For example, a web service might track how many:
+
+* Requests it received per second.
+* Milliseconds it took to respond.
+* Responses sent an error.
+
+These metrics can be reported to a monitoring system at regular intervals. If the web service is intended to respond to requests within 400 ms and starts responding in 600 ms, the monitoring system can notify the operations staff that the app response is slower than normal.
+
+## Using metrics
+
+There are two parts to using metrics in a .NET app:
+
+* **Instrumentation:** Code in .NET libraries takes measurements and associates these measurements with a metric name.
+* **Collection:** A .NET app configures named metrics to be transmitted from the app for external storage and analysis. Some tools may perform configuration outside the app using configuration files or a UI tool.
+
+Instrumented code can record numeric measurements, but the measurements need to be aggregated, transmitted, and stored to create useful metrics for monitoring. The process of aggregating, transmitting, and storing data is called collection. This tutorial shows several examples of collecting metrics:
```

Comment:
>Instrumented code can record numeric measurements [](http://example.com/codeflow?start=0&length=49)

do we need to mention the dimensions/tags here?

Reply:
> do we need to mention the dimensions/tags here?

@tarekgh can you suggest text for  dimensions/tags or provide a link?

Reply:
You may add the sentence `Measurements can also be associated with key-value pairs called tags that allow data to be categorized for analysis. ` and optionally add the link [Multi-dimensional metrics](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/metrics-instrumentation#multi-dimensional-metrics).